templates/crane.conf.erb

```diff
@@ -5,5 +5,5 @@
 [general]
 debug: false
 #data_dir_polling_interval: 60
-data_dir: /var/www/pub/docker/app
+data_dir: /var/lib/pulp/published/docker/v2/app/
```

Comment:
But if pulp still supports dockerv1 this doesn't make a ton of sense.  @rbarlow  thoughts?